Makefile

```diff
@@ -568,7 +572,13 @@ $(ROOTFS_IMG): $(ROOTFS_TAR) | $(INSTALLER)
 	        echo "ERROR: size of $@ is greater than 250MB (bigger than allocated partition)" && exit 1 || :
 	$(QUIET): $@: Succeeded
 
-$(LIVE).raw: $(BOOT_PART) $(EFI_PART) $(ROOTFS_IMG) $(CONFIG_IMG) $(PERSIST_IMG) | $(INSTALLER)
+$(LIVE).raw: $(LIVE_RAW_PARTS) | $(INSTALLER)
+	./tools/makeflash.sh -C 350 $| $@ $(PART_SPEC)
+	$(QUIET): $@: Succeeded
+
+$(LIVE).raw.imx8mq_evk: $(LIVE_RAW_PARTS) $(BSP_IMX_PART) | $(INSTALLER)
```

Comment:
I liked the way you did for the container to generate the raw image (add -t <platform>) . However, here I think that creating a new target for the imx platform can turn the build process more complex when we add more platforms. Don't you think would be better to use a variable (as it's done with HV and ZARCH) for the platform as well? Like this:

`make ZARCH=arm64 HV=kvm PLATFORM=imx8mq_evk live-raw`

This is exactly how it's done for OPTEE-OS and ATF. You can handle it on the $(LIVE).raw target or in the makeflash.sh script, any approach would work...

Reply:
I added the PLATFORM env, and changed the argument in pkg/eve runme.sh from -t <target> to -p <platform> for universality.